main/src/ca/uwaterloo/flix/language/ast/Symbol.scala

```diff
@@ -39,6 +39,7 @@ object Symbol {
   val Net: EffectSym = mkEffectSym(Name.RootNS, Ident("Net", SourceLocation.Unknown))
   val NonDet: EffectSym = mkEffectSym(Name.RootNS, Ident("NonDet", SourceLocation.Unknown))
   val Sys: EffectSym = mkEffectSym(Name.RootNS, Ident("Sys", SourceLocation.Unknown))
+  val Env: EffectSym = mkEffectSym(Name.RootNS, Ident("Env", SourceLocation.Unknown))
```

Comment:
Keep sorted

Reply:
If you do a usage search for Sys, do you find other places that it is used and where Env should be added? Or is it just here?